src/matrixroomhandler.ts

```diff
@@ -182,8 +182,8 @@ export class MatrixRoomHandler {
                     regexp: "[A-Za-z0-9_\-]{2,100}",
                 },
                 discriminator: {
-                    placeholder: "1234",
-                    regexp: "[0-9]{4}",
+                    placeholder: "0",
```

Comment:
Consider leaving the placeholder as `1234` to serve as a general example of a discriminator for a non-unique username.  (And once `0` becomes the only effective discriminator once Discord's username changes get fully rolled out, the bridge can stop caring about them entirely.)

Reply:
On some third party reimplementations such as spacebar (https://github.com/spacebarchat/server/pull/1049), the fact that discriminators exist is itself configurable.